incubator/mssql-linux/README.md

```diff
@@ -0,0 +1,126 @@
+# HELM Chart for Microsoft SQL Server 2017 on Linux
+
+## Prerequisites
+ * This chart requires Docker Engine 1.8+ in any of their supported platforms.
```

Comment:
This is a bit atypical - why does it have a docker engine version dependency?

Reply:
There was a prerequisite in the SQL Server documentation for 1.8 or higher.  I updated the README file with a link to the documentation.